docs/explanation/bgp-redirect.rst

```diff
@@ -0,0 +1,158 @@
+========================
+OVN integration with BGP
+========================
+
+OVN fully enabled support for integration with BGP in version ``25.03``. The
+support includes the ability to redirect BGP control-plane traffic to a
+specific ``Logical Switch Port`` and the ability of the ``ovn-controller`` to
+advertise and learn routes via a `Linux VRF`_.
+
+BGP control plane
+-----------------
+
+To avoid implementing the BGP protocol inside the OVN, the upstream project
+decided to provide a way to redirect the BGP control plane traffic from a
+``Logical Router Port`` to a ``Logical Switch Port``. As a result, an
+off-the-shelf BGP daemon can be run inside a VRF, bound to the ``Logical Switch
+Port``. Routing information is confined inside the VRF, leaving the routing
+table of the main host unaffected. And finally, the BGP daemon appears to the
+rest of the network as if it was bound to the ``Logical Router Port``.
+
+The last point is important for the purpose of hardware offload. If the BGP
```

Comment:
This detail of the implementation is also what allows "BGP unnumbered" to work, and it also allows the use of BGP authentication.